slack_api/slack_event_api_handler.py

```diff
@@ -25,34 +23,12 @@ class SlackEventApiHandler:
 
     def __init__(
             self,
-            slack_bot_token,
-            sftp_host,
-            sftp_username,
-            sftp_password,
-            sftp_port,
-            api_requester=SlackApiRequester,
-            sftp_navigator=SFTPNavigator,
```

Comment:
The Event Handler now imports an already requested navigator and requester, and the SFTP credentials are no longer passed to it.